app/editor/src/features/admin/topics/TopicList.tsx

```diff
@@ -59,35 +56,70 @@ export const TopicList: React.FC = () => {
     }
   }, [api, items.length, loading]);
 
+  React.useEffect(() => {
+    const mainElement = document.querySelector('main');
```

Comment:
Don't query the DOM, instead use a `useRef()`

Reply:
Updated, thanks